doc/codefmt.txt

```diff
@@ -70,7 +70,7 @@ The path to the google-java executable.  Generally, this should have the form:
 Default: 'google-java-format' `
 
                                                        *codefmt:shfmt_options*
-Command line arguments to to feed shfmt. Either a list or callable that takes
```

Comment:
This file is generated by running https://github.com/google/vimdoc over the source, e.g. from https://github.com/google/vim-codefmt/blob/ddc5b8/instant/flags.vim#L98. Could you fix the source and regenerate this file?

Reply:
Yes, I could do it.